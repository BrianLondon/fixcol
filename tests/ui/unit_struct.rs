use fixed_derive::{ReadFixed};

#[derive(ReadFixed)]
struct Thing;

pub fn main() {}
