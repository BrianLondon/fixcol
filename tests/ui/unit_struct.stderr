error: fixed-derive error: Cannot deserialize type with no inner data
 --> tests/ui/unit_struct.rs:4:8
  |
4 | struct Thing;
  |        ^^^^^
